src/components/TicketDetailModal/TicketDetailModal.tsx

```diff
@@ -0,0 +1,308 @@
+import {
```

Comment:
This component is overwhelming. I populated with buttons. But only title, description and delete button will be functioning. So I just care how it looks.

Reply:
I think you should split some parts of this component into sub-component. The name of this component is modal but considering the size, you can think of this as a kind of screen

Reply:
I would do that if I put all functionality for the buttons and everything.

Reply:
Why not now? It's a small team that I don't bother but if you do it in bigger team it's just making progress harder

Reply:
My intention of implementing here is only delete ticket and update ticket. You can feel free to do this if you like it to do.